<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="IR-C02PW2DWG8WP" name="innRoad.QAApiTesting.SuiteA.Test1" tests="3" failures="0" timestamp="27 Oct 2015 15:18:00 GMT" time="0.145" errors="1">
  <testcase name="test1" time="0.039" classname="innRoad.QAApiTesting.SuiteA.Test1">
    <error type="org.testng.SkipException" message="Skpping the test Test1 inside the suite SuiteA">
      <![CDATA[org.testng.SkipException: Skpping the test Test1 inside the suite SuiteA
	at innRoad.QAApiTesting.Base.TestBase.validateRunmodes(TestBase.java:42)
	at innRoad.QAApiTesting.SuiteA.Test1.test1(Test1.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
    </error>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.053" classname="innRoad.QAApiTesting.SuiteA.Test1"/>
  <testcase name="test1" time="0.053" classname="innRoad.QAApiTesting.SuiteA.Test1"/>
</testsuite> <!-- innRoad.QAApiTesting.SuiteA.Test1 -->
